miniforge3/tests/Dockerfile.root

```diff
@@ -0,0 +1,9 @@
+ARG IMAGE_TO_TEST=miniforge3
+FROM ${IMAGE_TO_TEST}
+
+COPY tests /tmp/tests
+RUN set -e; \
+    for f in /tmp/tests/*; do \
+        echo "Executing $f"; \
+        $f; \
+    done
```

Comment:
All the files here seem to be lacking a single newline at the end

Reply:
Fixed!